corehq/apps/accounting/management/commands/hide_invoices_by_id.py

```diff
@@ -24,7 +24,7 @@ def handle(self, *args, **options):
 
         for invoice_id in args:
             try:
-                invoice = Invoice.objects.get(pk=invoice_id)
+                invoice = super(InvoiceBaseManager, Invoice.objects).get_queryset().get(pk=invoice_id)
```

Comment:
wait so does this mean we have to use this syntax every time we want to get an invoice?